README.md

```diff
@@ -56,8 +56,14 @@ included will not be changed from the defaults.  `enabled` is set to false by de
 
 ### States
 
+#### `enableAll`
```

Comment:
We haven't yet updated the documentation to reflect the `enable`/`updateTheme` changes